biome.jsonc

```
@@ -110,18 +118,19 @@
         // This can likely be enabled for all non-utils files once these are eventually reworked, but until then we leave it off.
         "noExplicitAny": "off",
         "noAssignInExpressions": "off",
-        "noPrototypeBuiltins": "off",
+        "noPrototypeBuiltins": "off", // TOZO: consider enabling?
```

```suggestion
        "noPrototypeBuiltins": "off", // TODO: enable this
```
Now that we're on ES2023 it should be enabled.